articles/how-to-control-dark-mode-in-react-native-using-redux/index.md

```diff
@@ -0,0 +1,445 @@
+                           uri:
+                               'https://images.unsplash.com/photo-1573921470445-8d99c48c879f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1050&q=80',
+                       }}
+                   />
+                   <Paragraph>What is Lorem Ipsum?
+Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.
+i</Paragraph>
+               </Card.Content>
+           </Card>
+
+       </View>
+   )
+}
+export default Home
+```
+
+Here, we have used the Headline component to display the title. To create a card like component in the UI, we used the Card component with its subsidiary modules. Inside the Navigation.js file, we need to import the React Navigation components.
+- NavigationContainer
+- CreateBottomTabNavigator.
+
+We will get the result as displayed in the emulator screenshot below:
+
+![react-native-paper-in-home-screen.png](/engineering-education/how-to-control-dark-mode-in-react-native-using-redux/react-native-paper-in-home-screen.png)
+
+Next, we are going to implement the Setting screen. The Settings screen will be simple as it will only have a 'toggle component', that will allow us to toggle the theme modes within the app.
+
+In the Setting.js file, we need to add the code for the `Switch` component to be able to toggle over to a dark theme.
+
+The code for this is provided in the code snippet below:
+
+```
+import React from 'react';
+import { View, TouchableOpacity } from 'react-native';
+import {
+   List, Switch,
+} from 'react-native-paper';
+export default  ({ navigation }) => {
+
+    return (
+
+        <View style={{ flex: 1 }}>
+            <List.Item
+                title="Dark Mode"
+                left={() => <List.Icon icon="brightness-4" />}
+                right={() => <Switch />}
+            />
+        </View>
+    )
+}
+```
+
+We will get the result as displayed in the emulator screenshot below:
+
+![react-native-setting-screen.png](/engineering-education/how-to-control-dark-mode-in-react-native-using-redux/react-native-setting-screen.png)
+
+Here, we have added an icon as well as a switch component on the right. We are now done with the UI part. We can move on to the functionalities.
+
+### STEP 2: Setup React Navigation
+In order to set up the React Navigation, we need to re-structure our app a bit. First, we need to create a new file called `Navigation.js` at the root level that will contain the code configured for navigation purposes.
+
+Inside the Navigation.js file, we need to import the React Navigation components. They are `NavigationContainer` and `createBottomTabNavigator`. We also need to import other screens to which we need to navigate to.
```

Comment:
This seemed like it was repeating the steps above - it may be a completely new step but it was not very clear. The last sentence: "We also need to import other screens to which we need to navigate to." was also very vague - was not sure what other screens we are referring to.

Reply:
ok, I will replace other words with the actual screen name.